puppet/zulip_ops/manifests/profile/zmirror.pp

```diff
@@ -7,13 +7,11 @@
     'zephyr-clients',
     'krb5-config',
     'krb5-user',
-    # Packages needed to build pyzephyr
+    # Packages needed to for ctypes access to Zephyr
     'libzephyr-dev',
     'comerr-dev',
     'python3-dev',
```

Comment:
These development packages are unnecessary now, since we don’t build any C code. We reach directly into `libcom_err.so.2` and `libzephyr.so.4`, which are pulled in by `libzephyr4-krb5`.

Reply:
Updated. Do we need `python3-dev`?

Reply:
`python3-dev` is for building C extensions; we don’t need it.